infra/chroma/egress_handler/index.py

```
@@ -0,0 +1,28 @@
+
+
+def lambda_handler(event, _context):
+    url = os.environ.get("EXTERNAL_API_URL")
+    if not url:
+        return {
+            "statusCode": 200,
+            "body": json.dumps({"ok": True, "echo": event}),
+        }
+
+    try:
+        req = urllib.request.Request(url, method="GET")
```

Ruff (0.13.1)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)